charts/datadog-csi-driver/README.md

```diff
@@ -0,0 +1,19 @@
+# datadog-csi-driver
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+Datadog CSI Driver helm chart
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| fullnameOverride | string | `""` |  |
+| image.pullPolicy | string | `"IfNotPresent"` | CSI driver image pullPolicy |
+| image.pullSecrets | list | `[]` | CSI driver repository pullSecret (ex: specify docker registry credentials) |
+| image.repository | string | `nil` | Override default registry + image.name for CSI driver |
```

Comment:
Can you run `.github/helm-docs.sh`? That'll sync the values table with the descriptions from the `values.yaml` file and make the `helm-docs` ci job pass.